src/Page/SubmitStory/View.elm

```diff
@@ -0,0 +1,125 @@
+module Page.SubmitStory.View exposing (..)
+
+import Css exposing (Style, batch, displayFlex, flexGrow, flexShrink, flexWrap, hidden, int, overflow, pct, property, width, wrap)
+import Html.Styled exposing (Html, div, h2, iframe, img, li, p, text, ul)
+import Html.Styled.Attributes as Attr exposing (attribute, css)
+import I18n.Keys exposing (Key(..))
+import I18n.Translate exposing (Language(..), translate)
+import Message exposing (Msg)
+import Shared exposing (Model)
+import Theme.FluidScale
+import Theme.Global exposing (centerContent, listStyleNone, primaryHeader, roundedCornerStyle)
+
+
+view : Model -> Html Msg
+view model =
+    let
+        t : Key -> String
+        t =
+            translate model.language
+    in
+    div [ css [ centerContent ] ]
+        [ primaryHeader [] (t SubmitStoryHeading)
+        , submitFlex model
+        ]
+
+
+submitFlex : Model -> Html Msg
+submitFlex model =
+    let
+        t : Key -> String
+        t =
+            translate model.language
+    in
+    div
+        [ css
+            [ batch
+                [ displayFlex
+                , property "gap" "1rem"
+                , flexWrap wrap
+                ]
+            ]
+        ]
+        [ description (t SubmitStoryP1) (t SubmitStoryP2)
+        , form model
+        , images
+        ]
+
+
+description : String -> String -> Html Msg
+description p1 p2 =
+    div [ css [ flexChild ] ]
+        [ p [ css [ Theme.FluidScale.fontSizeMedium ] ] [ text p1 ]
+        , p [] [ text p2 ]
+        ]
+
+
+form : Model -> Html Msg
+form model =
+    let
+        formstack =
+            case model.language of
+                English ->
+                    iframe
+                        [ Attr.src "https://teamwildertest.formstack.com/forms/teamwilder_test"
+                        , Attr.title "teamwilder_test"
+
+                        -- these attributes are modified from those supplied by formstack
+                        , Attr.attribute "width" "100%"
+                        , Attr.height 1200
+                        ]
+                        []
+
+                Welsh ->
+                    -- this would need to be a separate welsh form
+                    iframe
+                        [ Attr.src "https://teamwildertest.formstack.com/forms/teamwilder_test"
```

Comment:
If we move the src and title into I18n Keys, then we won't need these defined separately like this here.